build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -152,6 +239,25 @@ service ActionCache {
     option (google.api.http) = { get: "/v2/{instance_name=**}/actionResults/{action_digest.hash}/{action_digest.size_bytes}" };
   }
 
+  // Retrieves a set of potential matches for a weak fingerprint. If a
```

Comment:
>weak fingerprint [](start = 48, length = 16)

I'm not sure the build community out there knows what weak/strong fingerprints are. I'd rather use the term command fingerprint and observed input digest.

Reply:
Command fingerprint does not make sense as there are '"Command" and "Action" data structures in the specification, and what we are referring to is a weak action fingerprint. Change weak fingerprint to "action fingerprint" across the spec.

---
In reply to: [458991316](https://github.com/erikma/remote-apis/pull/1#discussion_r458991316) [](ancestors = 458991316)

Reply:
Updated mentions of strong fingerprint and prospective strong fingerprint to "observed-input digest" including field names. Good choice in terminology, reading flows better with that naming.

---
In reply to: [460338076](https://github.com/erikma/remote-apis/pull/1#discussion_r460338076) [](ancestors = 460338076,458991316)